Rakefile

```diff
@@ -71,6 +77,91 @@ def re_run_basic_smoke_test
   sh beaker
 end
 
+def jenkins_passing_json_parsed
```

Comment:
These look to be duplicating https://github.com/puppetlabs/puppetserver/blob/master/acceptance/lib/helper.rb#L290-L312 , maybe we should coalesce these helper methods.